api/models.py

```diff
@@ -7,6 +7,7 @@
 
 from typing import Optional, Dict
 
+from datetime import datetime
```

Comment:
```suggestion
from datetime import datetime
from typing import Optional, Dict

```
The imports should be grouped by category: first the ones from the standard Python library, then additional packages and finally local ones from the current package. We don't strictly need blank lines but it helps making the distinction. Also, keeping them in alphabetical order avoids adding duplicates and it's easier to maintain.